core/templates/dev/head/pages/exploration_player/information_card_modal_directive.html

```diff
@@ -8,9 +8,10 @@ <h2 class="oppia-info-card-title"><[explorationTitle]></h2>
     <p ng-if="objective">
       <[objective | truncateAndCapitalize: 95]>
     </p>
-    <p ng-if="!objective" translate="I18N_PLAYER_NO_OBJECTIVE">No objective specified.</p>
+    <p ng-if="!objective && !isExplorationPrivate" translate="I18N_PLAYER_NO_OBJECTIVE">No objective specified.</p>
+    <p ng-if="!objective && isExplorationPrivate" class="private-card-info" translate="I18N_PLAYER_IS_PRIVATE">This exploration is private.</p>
```

Comment:
This should not be conditioned on the absence of an objective. It should be shown in the area below that, where the rating stars and share buttons were initially placed.

Reply:
I didn't quite get this...
In your previous review you suggsted that "Let's keep the objective/no-objective line in both cases (public and private)." I misunderstood it?

Reply:
Yes, you did. The objective line should be shown in all cases. The part after that should change depending on the exploration's status.

Reply:
Oh got it! Just to be clear, this is what you meanr right?
![screenshot from 2018-03-03 14-23-31](https://user-images.githubusercontent.com/20617436/36932428-9a98044a-1eee-11e8-859c-4a23c609e86a.png)

Reply:
Yes, but please left-align, change "The" to "This", and de-capitalize "Exploration".

Reply:
okay done!